deepmd/pt/model/task/fitting.py

```diff
@@ -388,14 +388,22 @@ def get_dim_aparam(self) -> int:
         """Get the number (dimension) of atomic parameters of this atomic model."""
         return self.numb_aparam
 
+    # make jit happy
+    exclude_types: List[int]
```

Comment:
could you plz check if jit passes if we change the line 
https://github.com/deepmodeling/deepmd-kit/blob/d81443ba37adab034d7ffa9e0523b141cd96a1c4/deepmd/pt/model/task/fitting.py#L329
to 
```python
self.exclude_types : List[int] = exclude_types
```

Reply:
xref: https://github.com/pytorch/pytorch/pull/67051